charts/partials/templates/_proxy.tpl

```diff
@@ -5,7 +5,7 @@ env:
   value: "{{.Values.global.proxy.requireIdentityOnInboundPorts}}"
 {{ end -}}
 - name: LINKERD2_PROXY_LOG
-  value: {{.Values.global.proxy.logLevel}}
+  value: "{{.Values.global.proxy.logLevel | default "linkerd=info,warn" }}"
```

Comment:
The default for this value from the chart is already "warn,linkerd=info" so this value should never be empty at this point.  I think having defaulting behavior here also just confuses things.  (for what it's worth, that's also true about the defaulting behavior below in the log format variable and probably elsewhere throughout these templates)